test/views/utils/ViewUtilsSpec.scala

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.utils
+
+import base.SpecBase
+import generators.Generators
+import models.departure.{NoReleaseForTransitMessage, ResultsOfControl}
+import org.scalacheck.Arbitrary.arbitrary
+import org.scalacheck.Gen
+import org.scalatestplus.scalacheck.ScalaCheckPropertyChecks
+import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
+import views.utils.ViewUtils._
+
+class ViewUtilsSpec extends SpecBase with ScalaCheckPropertyChecks with Generators {
+
+  "RichFunctionalError" - {}
```

Comment:
Is this to be completed later or accidentally left in?

Reply:
Good spot! Forgot to add that in - will do it now